srcpkgs/python3-aiohttp/template

```diff
@@ -1,21 +1,25 @@
 # Template file for 'python3-aiohttp'
 pkgname=python3-aiohttp
-version=3.7.4
-revision=5
+version=3.9.0b1
+revision=1
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-wheel"
 makedepends="python3-devel http-parser-devel"
-depends="python3-async-timeout python3-attrs python3-chardet python3-yarl
- python3-typing_extensions python3-multidict"
+depends="python3-aiosignal python3-async-timeout python3-attrs
+ python3-chardet python3-yarl python3-typing_extensions python3-multidict"
+checkdepends="$depends python3-async_generator python3-Brotli
+ python3-charset-normalizer python3-freezegun python3-pytest
+ python3-pytest-cov python3-pytest-forked python3-pytest-mock
+ python3-pytest-timeout python3-pytest-xdist"
 short_desc="HTTP client/server for asyncio (PEP-3156)"
 maintainer="Michael Aldridge <maldridge@voidlinux.org>"
 license="Apache-2.0"
 homepage="https://aiohttp.readthedocs.io/"
 distfiles="${PYPI_SITE}/a/aiohttp/aiohttp-${version}.tar.gz"
-checksum=5d84ecc73141d0a0d61ece0742bb7ff5751b0657dab8405f899d3ceb104cc7de
+checksum=d6e120b08ac168825239c64e0a850a108edb9cd17be247e25bced9b07a14a403
+make_check=no # Depends on proxy.py and re_assert
```

Comment:
If tests are disabled here because they require dependencies not packaged for Void, the entire list of `checkdepends` should be dropped.